### ISSUE TITLE
Why is NeighborRateRatio not considered when measuring PTP path delay

### ISSUE THREAD

**Issue by liu-jeff** [ptpd/ptpd#91]
Hello, I am very sorry to bother you , but I have some confusion about this, why is NeighborRateRatio not considered when measuring PTP path delay.
![54bd2941-83ee-4605-bf2c-94543bb4880b](https://user-images.githubusercontent.com/50905379/185556921-41aaef45-7203-4917-9e98-2a6d753b0093.jpeg)

**Comment by aaeegg** [comment]
ptpd calculates meanPathDelay according to IEEE 1588-2008 11.4.3. The equation in IEEE 1588-2008 11.4.3 doesn't include neighbor rate ratio correction. IEEE 1588-2008 6.5.5 offers neighbor rate ratio correction of link delay as a way to reduce residence time inaccuracy in peer-to-peer transparent clocks, but doesn't require it, and doesn't suggest such correction on ordinary clocks. On the other hand, the equation in IEEE Std 802.1AS-2020 16.4.3.2 includes neighbor rate ratio correction in the mean link delay calculation. IEEE Std 802.1AS-2020 seems to strongly suggest that all PTP instances should perform neighbor rate ratio correction, although 12.5.2.4.4 allows "other means" of computing mean link delay.

So in short, I believe ptpd disregards neighbor rate ratio because ptpd implements IEEE 1588-2008 PTPv2 and not IEEE Std 802.1AS-2020 gPTP.
